docs/increase-performance.rst

```diff
@@ -0,0 +1,67 @@
+.. _increase-performance:
+
+==============================================================
+Increase Locust's performance with a faster HTTP client
+==============================================================
+
+Locust's default HTTP client uses `python-requests <http://www.python-requests.org/>`_. 
+The reason for this is that requests is a very well-maintained python package, that 
+provides a really nice API, that many python developers are familiar with. Therefore, 
+in many cases, we recommend that you use the default :py:class:`HttpLocust <locust.core.HttpLocust>` 
+which uses requests. However, if your're planning to run really large scale scale tests, 
+Locust comes with an alternative HTTP client, 
+:py:class:`FastHttpLocust <locust.contrib.fasthttp.FastHttpLocust>` which 
+uses `geventhttpclient <https://github.com/gwik/geventhttpclient/>`_ instead of requests.
+This client is significantly faster, and we've seen 5x-6x performance increases for 
```

Comment:
I'm hesitant to make any specific performance claims since so many factors are in play.  I'd probably reword to something like:
"This client is significantly faster, and we've seen significant performance improvements for many workloads"

Reply:
I've made it clearer (I think) in: 42bfffb3d8eaef8205ccc4f7cdead9dd9a22c2ae